Added shape painter option to Multitouch Joystick extension

tristanbob opened the pull request:
## Updates

- Added behavior for shape painter (Thanks Melon!)
- Changed wording from "Activate joystick" to "Animate joystick"
- No change to function names to prevent impacting existing games

## Playable game
https://liluo.io/games/968717d9-209e-4a9d-a5e5-f6ec995161cb
Works with touch, mouse (sprite=left-click, painter=right-click) and gamepad inputs (sprite=controller 1, painter=controller 2)

## Project files
[MultitouchJoystick Shape Painter.zip](https://github.com/GDevelopApp/GDevelop-extensions/files/8945118/MultitouchJoystick.Shape.Painter.zip)

tristanbob commented:
Is anyone interested in approving this change?  Thanks!

tristanbob commented:
Hmmmm... how to resolve the conflict?  I think it was caused by moving the extensions into community/reviewed folders.  

1) I could ```Merge branch "Main" into this branch"```

or

2) Close this PR and create a new PR based on current version of "Main"

Bouh commented:
- The description don't talk about the shape painter
- 

> Hmmmm... how to resolve the conflict? I think it was caused by moving the extensions into community/reviewed folders.
> 
> 1. I could `Merge branch "Main" into this branch"`
> 
> or
> 
> 2. Close this PR and create a new PR based on current version of "Main"

Like you prefer both works.

Here a short review:
- The description of the extension don't talk about the shape painter and how to use the extension with it.
- `shape painter` can be added in tags.

Bouh reviewed (changes requested):
- The description of the extension don't talk about the shape painter and how to use the extension with it.
- `shape painter` can be added in tags.

tristanbob commented:
> * The description of the extension don't talk about the shape painter and how to use the extension with it.

I added this to the description.

> * `shape painter` can be added in tags.

Added.

tristanbob commented:
Thanks for the good input, @Bouh!  Here are the updated files, if needed.

## Playable game

https://liluo.io/victrisgames/multitouchjoystick-shape-paint

## Project files

[MultitouchJoystick Shape Painter.zip](https://github.com/GDevelopApp/GDevelop-extensions/files/9483599/MultitouchJoystick.Shape.Painter.zip)

tristanbob commented:
Reminder to Tristan: check if another PR was created for this.

Bouh commented:
Now we have joystick object in the object list.
Is that ok to close this PR?

4ian commented:
I think so yes?
